spec/decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './../modules/decorator'
+require_relative './../modules/nameable'
+
+describe '#new' do
+  decorator = Decorator.new('label')
+  it 'takes a new decorator object' do
+    expect(decorator).to be_instance_of(Decorator)
+  end
+
+  it 'returns a nameable object' do
```

Comment:
- [ ] Kindly make sure to add unit tests for all the methods of the: `Decorator class` ✍🏻 Cureent you're missing the `correct_name` method tests. 👇🏻  


```ruby
  def correct_name
    @nameable.correct_name
  end
```